package/boot/uboot-envtools/files/ath79

```diff
@@ -41,6 +41,7 @@ engenius,ecb1750|\
 engenius,ecb350-v1|\
 engenius,ecb600|\
 engenius,enh202-v1|\
+engenius,ens1750|\
```

Comment:
You can add a `compatible = "u-boot,env";` property to the device tree u-boot-env partition. This file is not necessary now. ref: https://github.com/openwrt/openwrt/commit/d7d94a8d91bc1dc9dcedbe004021509871acac24

Reply:
Done (for all devices using qca955x_senao_loader.dtsi). Left as a separate commit, let me know if it should be it's own PR due to the wide-reaching change.